ibm/mas_devops/common_vars/casebundles/v8-airgap-amd64.yml

```diff
@@ -9,9 +9,14 @@ common_svcs_version: 1.19.4   # Operator version 3.23.4 (https://github.com/IBM/
 db2u_version: 5.1.4           # Operator version 110508.0.2 (https://github.com/IBM/cloud-pak/tree/master/repo/case/ibm-db2uoperator)
 events_version: 4.6.1         # Operator version 4.6.1 (https://github.com/IBM/cloud-pak/tree/master/repo/case/ibm-events-operator)
 uds_version: 2.0.11           # Operator version 2.0.11
-sls_version: 3.7.0-pre.stable # Operator dev version
-tsm_version: 1.5.1-pre.stable # Operator dev version
+sls_version: 3.8.0-pre.stable # Operator dev version
+tsm_version: 1.5.2-pre.stable # Operator dev version
```

Comment:
I'd suggest sticking with tsm_version: 1.5.1-pre.stable or 1.5.1, Truststore manager hasn't been published at 1.5.1 yet, so there shouldn;t be a v1.5.2-pre-stable yet

Reply:
Oh i see. i saw that fvtcpd was installing it so I thought it was ready. I will make that change. Thank you